shared/overview/2.x.md

```diff
@@ -125,26 +128,37 @@ Now for the final piece of the puzzle, the board-specific meta-{{ $names.company
-|-----------------------------------|-------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|
-| meta-{{ $names.company.lower }}                        | {{ $links.githubOS }}/meta-balena                                        | This repository enables building {{ $names.os.lower }} for various devices                         |
-| meta-{{ $names.company.lower }}-jethro                 | {{ $links.githubOS }}/meta-balena                                        | This layer enables building {{ $names.os.lower }} for jethro supported BSPs                        |
-| meta-{{ $names.company.lower }}-raspberrypi            | {{ $links.githubOS }}/{{ $names.company.lower }}-raspberrypi                                 | Enables building {{ $names.os.lower }} for chosen meta-raspberrypi machines.                        |
-| meta-raspberrypi                  | https://github.com/agherzan/meta-raspberrypi                                  | This is the general hardware specific BSP overlay for the Raspberry Pi device family. |
-| meta-openembedded                 | http://git.openembedded.org/meta-openembedded                                 | Collection of OpenEmbedded layers                                                     |
-| meta-openembedded/meta-oe         | https://github.com/openembedded/meta-openembedded/tree/master/meta-oe         |                                                                                       |
-| meta-openembedded/meta-python     | https://github.com/openembedded/meta-openembedded/tree/master/meta-python     | The home of python modules for OpenEmbedded.                                          |
-| meta-openembedded/meta-networking | https://github.com/openembedded/meta-openembedded/tree/master/meta-networking | Central point for networking-relatedpackages and configuration.                       |
-| oe-meta-go                        | https://github.com/mem/oe-meta-go                                             | OpenEmbedded layer for the Go programming language                                    |
-| poky/meta-yocto                   | https://git.yoctoproject.org/cgit/cgit.cgi/meta-yocto/                                   |                                                                                       |
-| poky/meta                         | https://git.yoctoproject.org/cgit/cgit.cgi/poky/                                         | Core functionality and configuration of Yocto Project                                 |
+| Layer Name                                  | Repository                                                                    | Description                                                                           |
```

Comment:
Looks like my editor formatted this table. I know this is hugely out of date but please don't review this section (unless you have a strong opinion on markdown table formatting)  as I'll do a separate Yocto PR.